room.go

```diff
@@ -33,6 +33,7 @@ type ConnectInfo struct {
 type ConnectParams struct {
 	AutoSubscribe bool
 	Reconnect     bool
+	Callback      *RoomCallback
```

Comment:
hmm, now I'm thinking it might be a better idea to take this in as an explicit argument in `ConnectToRoom`, users cannot leave this blank.